federator/src/lib/TransactionSender.ts

```diff
@@ -44,17 +44,21 @@ export class TransactionSender {
   }
 
   async getGasLimit(rawTx) {
-    const estimatedGas = await this.client.eth.estimateGas({
-      gasPrice: rawTx.gasPrice,
-      value: rawTx.value,
-      to: rawTx.to,
-      data: rawTx.data,
-      from: rawTx.from,
-    });
-    // Gas estimation does not work correctly on RSK and after the London harfork, neither is working on Ethereum
-    // example https://etherscan.io/tx/0xd30d6cf428606e2ef3667427b9b6baecb2f4c9cbb44a0c82c735a238ec8f72fb
-    // To fix it, we decided to use a hardcoded gas estimation
-    return +estimatedGas < ESTIMATED_GAS ? ESTIMATED_GAS : +estimatedGas;
+    try {
+      const estimatedGas = await this.client.eth.estimateGas({
+        gasPrice: rawTx.gasPrice,
+        value: rawTx.value,
+        to: rawTx.to,
+        data: rawTx.data,
+        from: rawTx.from,
+      });
+      // Gas estimation does not work correctly on RSK and after the London harfork, neither is working on Ethereum
+      // example https://etherscan.io/tx/0xd30d6cf428606e2ef3667427b9b6baecb2f4c9cbb44a0c82c735a238ec8f72fb
+      // To fix it, we decided to use a hardcoded gas estimation
+      return +estimatedGas < ESTIMATED_GAS ? ESTIMATED_GAS : +estimatedGas;
+    } catch (err) {
+      return null;
```

Comment:
This looks dangerous,  as there is an actual error here and it's being swallowed